bokun.yaml

```diff
@@ -5141,118 +5142,179 @@ definitions:
         format: double
   ActivityAvailabilityDto:
     properties:
+      id:
+        type: string
+        description: 'Value "startTimeId_YYYYMMDD" (YYYYMMDD - date of the departure for this particular availability).'
       activityId:
         type: integer
         format: int64
+        description: 'Id of the activity'
+      activityTitle:
+        type: string
+        description: 'Title of the activity.'
       activityOwnerId:
         type: integer
         format: int64
+        description: 'Id of vendor which owns activity (original supplier)'
+      activityOwnerTitle:
+        type: string
+        description: 'Title of vendor which owns activity (original supplier)'
+      startTime:
+        type: string
+        description: 'Start time in hours:minutes using 24h format (e.g. "19:00"). If this is not a time based availability, e.g. pass type, then value is "00:00"'
+      startTimeId:
+        type: integer
+        format: int64
+        description: 'Unique system id for the start time mapped to this availability'
+      startTimeLabel:
+        type: string
+        description: 'Optional label if used for this start time'
+      flexible:
+        type: boolean
+        description: 'If activity "bookingType" is DATE then true, otherwise - false.'
+      date:
+        type: integer
+        format: int64
+        description: 'Timestamp for request current date (at time exactly 0:00 UTC).'
+      localizedDate:
+        type: string
+        description: 'Current date when request was sent in profile language (e.g. "Wed 18.Mar 20").'
       availabilityCount:
         type: integer
         format: int32
+        description: 'Amount of open seats for this start time. Returns 1000 if unlimited.'
```

Comment:
I'd not specifically mention 1000 because people might start depending on it. Something along the lines of `returns arbitrary high value if unlimited`